src/scaler/spreadscaler/link.rs

```diff
@@ -285,6 +295,12 @@ impl<S: ReadStore + Send + Sync, L: LinkSource> LinkScaler<S, L> {
     }
 }
 
+fn compute_linkscaler_values_hash(values: &HashMap<String, String>) -> u64 {
+    let mut linkscaler_values_hasher = std::collections::hash_map::DefaultHasher::new();
+    BTreeMap::from_iter(values.iter()).hash(&mut linkscaler_values_hasher);
+    linkscaler_values_hasher.finish()
+}
```

Comment:
@thomastaylor312 you've done more in-depth hashing things in Rust than I, does this look like a good approach to getting a hash of the map?

Reply:
I think for the purposes we're using it for here, this works great